src/main/java/duke/logic/parsers/ParserUtil.java

```diff
@@ -30,6 +53,130 @@ protected static Todo createTodo(String userInput) throws DukeException {
         return new Todo(description);
     }
 
+    /**
+     * Parses the userInput and return a new Event constructed from it.
+     *
+     * @param userInput The userInput read by the user interface.
+     * @return The new Event object.
+     */
+    protected static Event createEvent(String userInput) throws DukeException {
```

Comment:
i throw a few of this methods to the parser itself already cause theres no other uses for it

Reply:
Ok then i will delete the duplicate ones